swift.go

```diff
@@ -1963,7 +1967,7 @@ func (c *Connection) doBulkDelete(objects []string) (result BulkDeleteResult, er
 // See also:
 // * http://docs.openstack.org/trunk/openstack-object-storage/admin/content/object-storage-bulk-delete.html
 // * http://docs.rackspace.com/files/api/v1/cf-devguide/content/Bulk_Delete-d1e2338.html
-func (c *Connection) BulkDelete(container string, objectNames []string) (result BulkDeleteResult, err error) {
+func (c *Connection) BulkDelete(container string, objectNames []string, h Headers) (result BulkDeleteResult, err error) {
```

Comment:
This breaks the public interface of the library, alas.

You'll have to rename this one `BulkDeleteHeaders` and get `BulkDelete` to call it.

Reply:
Ok! Will update the PR tomorrow!